docs/reference/authentication.rst

```diff
@@ -1,8 +1,9 @@
-.. _authenticate:
+.. _api-authentication:
+
+==================
+API Authentication
+==================
 
-==============
-Authentication
-==============
 
 The Flocker REST API uses TLS to secure and authenticate requests.
```

Comment:
thanks Michael. I think we can define and publish the terminology when we reach the point of actually adding the tagging to the docs. Otherwise, this particular branch is going to prove too difficult to reach the point of merging. I'll keep the issue active, but remove the "prio1" tag for the time being.